src/components/explore-projects/exploreProjectsUtils.js

```diff
@@ -0,0 +1,119 @@
+const setStyesOnProjectImage = (styles, element, e) => {
+  Object.assign(element.style, styles);
+};
+const handleHoverLogic = (
+  e,
+  onHoverClass,
+  noneHoveredProjectImagesStyle,
+  setLayoutVisible
+) => {
+  const imagesList = e.target.parentNode.parentNode.children;
+  [...imagesList].map((imageList) => {
```

Comment:
esLint error: Array.prototype.map() expects a value to be returned at the end of arrow function.

Reply:
you also have couple of esLint errors to handle, check them out